mc_providers/mediacloud.py

```diff
@@ -175,12 +176,18 @@ def _query(self, endpoint: str, params: Dict = None, method: str = 'GET'):
         if params and ('q' in params):
             params['q'] = sanitize_query(params['q'])
         endpoint_url = self.API_BASE_URL+endpoint
+        start = time.time()
```

Comment:
Tiny comment: `time.monotonic()` returns TRUE elapsed time (`time.time()` is effected by system clock changes/adjustments) -- need to use it in both places.  And since you're collecting it, maybe log the time? with the params?? (maybe at debug level if that's too much output???)